code/game/objects/items/toys/toys_ch.dm

```diff
@@ -22,4 +22,23 @@
     name = "green fox plushie"
     icon = 'icons/obj/toy_ch.dmi'
     icon_state = "greenfox"
-    pokephrase = "Weh!"
\ No newline at end of file
+    pokephrase = "Weh!"
+
+/obj/item/toy/plushie/red_dragon
+	name = "red dragon plushie"
+	icon = 'icons/obj/toy_ch.dmi'
+	icon_state = "reddragon"
+	pokephrase = "RAWR!"
+
+/obj/item/toy/plushie/green_dragon
+	name = "green dragon plushie"
+	icon = 'icons/obj/toy_ch.dmi'
+	icon_state = "greendragon"
+	pokephrase = "GRR!"
+
+/obj/item/toy/plushie/purple_dragon
+	name = "red dragon plushie"
```

Comment:
red twice?

Reply:
Ah, thanks for pointing that out! guess I missed it qwq